abyss.c

```diff
@@ -116,9 +117,7 @@ void kill_all_levels()
     kill_levels("om*.lev");
 }
 
-void kill_levels(str)
```

Comment:
Looks good to me, and the new standard is what I'm more familiar with anyway when working in C++